packages/tpetra/core/src/Tpetra_CrsGraph_decl.hpp

```diff
@@ -2151,6 +2182,10 @@ namespace Tpetra {
     ///
     void computeLocalConstants();
 
+    /// \brief Forces computation of local triangular properties if they have
+    /// not been computed yet.
+    void computeTriangularProperties();
```

Comment:
I would call this "computeLocalTrangularProperties" just to make clear that it's local.